src/languages/elixir.md

```diff
@@ -0,0 +1,135 @@
+# Elixir
+
+Platform.sh supports building and deploying applications written in Elixir.  They are automatically compiled during the Build phase, and support both committed dependencies and download-on-demand. The underlying Erlang version is 
+22.0.7.
+
+## Supported versions
+
+* 1.9
+
+To specify an Elixir container, use the `type` property in your `.platform.app.yaml`.
+
+```yaml
+type: 'elixir:1.9'
+```
+
+## Assumptions
+
+Platform.sh is making assumptions about your application to provide a more streamlined experience. These assumptions are the following:
+
+- Your `mix.exs` file is at the root of your application.
+
+Platform.sh will then run `mix do deps.get, deps.compile, compile` on your application to build a binary.
+
+If you don't want these assumptions, you can disable this behavior by specifying in your `.platform.app.yaml`:
+
+```yaml
+build:
+    flavor: none
+```
+
+## Dependencies
+
+The recommended way to handle Elixir dependencies on Platform.sh is using Hex. Commit a `mix.exs` file in your repository and the system will automatically download the dependencies in you `deps` section.
+
+```elixir
+  defp deps do
+    [
+	  {:platformshconfig, "~> 0.1.0"}
+    ]
+  end
+```
+
+
+## Platform.sh variables
+
+Platform.sh exposes relationships and other configuration as [environment variables](/development/variables.md). Most notably, it allows a program to determine at runtime what HTTP port it should listen on and what the credentials are to access [other services](/configuration/services.md).
```

Comment:
This is redundant with the rest of the docs.

Reply:
OK We can just keep the note about when stuff is evaluated..